### DISCUSSION TITLE
Value Error Cannot Sign

### DISCUSSION THREAD

**Discussion by andretrevisani** [selfcustody/krux#315] [General]
Hey team, I am trying to create a timelocked TX but while I can load it, I can't sign it. The system tells me `Value Error Cannot Sign`.
I am using Electrum to create it. I also have tried with Specter but I can't find a way to add a timelock there.

I created a sample on the testnet:

- **Mneumonic** (no pass): nice ghost era idea clip rival update siren shy acquire arctic reward
- **Vpub**: vpub5YD3pgwZGvmux1DbPvXxGWenjphaSc5qiTQpyGMQhVhK6W4DsGRLJqHp5dUqmYFLW2GXFm8tQk11fMzyoMKWribumi6CEUEnqgaWTi9yVT6

**PSBT**
```
cHNidP8BAFICAAAAAU57k4BBUrHgxj8HeAmwPY4vKzvDfUVWGRr0D92M+0quAQAAAAD9////AWRjAQAAAAAAFgAUb43zLEn1QY5NkAn1TD131lzcuZjly3RnAAEBH9JjAQAAAAAAFgAUQtQdAp0SIMoRJ0tdwvWbQVp8l04BAN4CAAAAAAEBIypLwoKiQC2fOQfNF91BdGjl5l2lpXg7QZMUezYpm1wAAAAAAP3///8CDdLV4QAAAAAWABSx7FGm0R2UbRyW7yErTSrK0s8XytJjAQAAAAAAFgAUQtQdAp0SIMoRJ0tdwvWbQVp8l04CRzBEAiB5VJOJjxDqp+dO6QtHM2pHuTiipd/tzu5vNvVhSuPSbwIgOyhDsIF64UX72b91uekvdgjZXm9ncCJWR1AGB8T2CoIBIQP00JBKw7WYlc3K/3jkTucP+SuSzSvCqpFLR35V6kJw4QhWJwAiBgJGjw9ZGsI614FVxRW+VNn8EG1AIiwHjGFDT0KtaCKUIwxRP+UoAAAAAAAAAAAAIgIDyDnzs8jZCVg9GJz15q8M4Zc+HCoZiWVXet/dTHr3ohMMUT/lKAAAAAABAAAAAA==
```

Here is a 10-part PSBT
[PSBT-parts.zip](https://github.com/selfcustody/krux/files/14250948/PSBT-parts.zip)

System is able to load it
![tx-confirm](https://github.com/selfcustody/krux/assets/2761413/e9ed5209-8106-445e-99fc-d17a4f626a67)

But when trying to sign it errors out
![tx-error](https://github.com/selfcustody/krux/assets/2761413/f21d2ce7-2fd3-4b0c-9f42-623df4077b64)

Here is the Electrum UI, note the locktime to 2025/01/01
![tx-ui](https://github.com/selfcustody/krux/assets/2761413/6b7cbc7e-a438-4465-8d0e-2f4dda5a590b)

I can create and sign non-timelocked TXs on Specter. I also imported this TX to Specter, and then exported it from Specter to load and sign, also without success.

**Comment by odudex** [comment]
Hi André, unfortunately Krux is not compatible with Electrum. On PC you can use Specter and Sparrow. I recommend you to import the Vpub from Krux to one of these two coordinators, then create the transaction from them. Krux exported Vpub contains its fingerprint, that will be later used on PSBT signing for extra verifications.

**Comment by tadeubas** [comment]
I think the timelock feature on Specter Desktop is visible only on advanced mode, plz see this issue for more info: https://github.com/cryptoadvance/specter-desktop/issues/1101
